src/registry/SsiAuthoritySchemaRegistry.Service/Controllers/SchemaController.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.AspNetCore.Mvc;
+using Org.Eclipse.TractusX.SsiAuthoritySchemaRegistry.Service.BusinessLogic;
+using Org.Eclipse.TractusX.SsiAuthoritySchemaRegistry.Service.Extensions;
+using Org.Eclipse.TractusX.SsiAuthoritySchemaRegistry.Service.Models;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json;
+
+namespace Org.Eclipse.TractusX.SsiAuthoritySchemaRegistry.Service.Controllers;
+
+public static class SchemaController
+{
+    [ExcludeFromCodeCoverage]
```

Comment:
non-testing code should not be excluded from coverage

Reply:
removed